### ISSUE TITLE
Milestone 3-Submission

### ISSUE THREAD

**Issue by tuhoag** [browne0/FactsOfToday#3]
I do not see your schema. Please make sure you have pushed all your latest project files to your GitHub Repo.

**Comment by browne0** [comment]
We have included our Data Schema in our README.md

Sorry for not making that clearer in the document.

**Comment by tuhoag** [comment]
Hi @browne0, did you mean the information in APIs section?

**Comment by browne0** [comment]
Yes @hatu87 , unless we are doing that wrong that is our data schema.

**Comment by knofte** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

🎉 Congratulations! You've won a share of the $1.500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. But that's not all—connect now, and you'll instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://link.bnbreward.online/0mhPJ2)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](http://link.bnbreward.online/0mhPJ2) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1.500k in BNB! Plus, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, seize the chance to win up to $20,000—it's your golden opportunity!


Winners: @snehal-das, @npmarrin, @amlcodes, @itsabh1sh3k, @gui64, @ArthurOstapenko, @felipeccastro, @kevin-chau, @sharad-s, @tsuilouis, @rmrunltd, @xwartz, @markcapaldi, @lceames, @clongbupt, @shumailxyz, @ericxtang, @abcijkxyz, @basnetaakash23, @mvaz, @shyamsalimkumar, @helloqinglan, @TaylorDennisLee, @mirocosic, @DiabloDaN, @andrijac, @bichenkk, @soundslocke, @isitinme, @samuelcesc, @mviamari, @MarkoCen, @apires89, @qdriven, @mishushakov, @bonedaddy, @anka09, @DanielBodnar, @admiralrohan, @mattlinares, @mkobar, @DiegoAscanio, @phatpham9, @skyjia, @xukeek, @erik-sjoestedt, @chipz, @kustov-an, @simonchan2013, @brunobertolini
